.circleci/main.yml

```diff
@@ -318,10 +318,10 @@ jobs:
             - v1-http-client-{{ checksum "~/ipfs/go-ipfs/go-ipfs-http-client/go.sum" }}
             - v1-http-client-
       - run:
-          name: go test -v ./...
+          name: go test -a -v ./...
           command: |
             export PATH=/tmp/circleci-workspace/bin:$PATH
-            go test -v ./...
+            go test -a -v ./...
```

Comment:
Nit: consider using `count=1` instead of `-a` which will rerun w/o rebuilding, and the docs bless as idiomatic:

> The idiomatic way to disable test caching explicitly is to use -count=1.